lib/Cache/RuntimeCacheTrait.php

```diff
@@ -46,6 +52,7 @@ public static function getCacheEnabled(): bool
      *
      * @param mixed $config
      * @param string $cacheKey
+     * @deprecated
```

Comment:
we should trigger a deprecation warning in methods like:
```
trigger_deprecation(
    'pimcore/pimcore',
    '10.5.0',
    sprintf('%s is deprecated and will be removed in Pimcore 11. Use \Pimcore\Cache\Runtime::set() instead.', __METHOD__)
);
```

Reply:
Thanks! Will change this accordingly!

Reply:
`RuntimeCacheTrait` has been deprecated with deprecation warnings